server.js

```diff
@@ -13,10 +14,10 @@ new WebpackDevServer(webpack(config), {
   stats: {
     colors: true,
   },
-}).listen(4242, 'localhost', function (err) {
+}).listen(port, 'localhost', function (err) {
   if (err) {
     console.log(err);
   }
 
-  console.log('Listening at localhost:4242');
+  console.log(`Listening at localhost:${port}`);
```

Comment:
So, putting the log here is too late: if the port's already used, you'll have the following:

```
yarn start v0.24.6
$ mkdir -p dist && cp res/zee-worker.js dist/ && NODE_ENV=development node server.js 
events.js:182
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE 127.0.0.1:4242
    at Object.exports._errnoException (util.js:1016:11)
    at exports._exceptionWithHostPort (util.js:1039:20)
    at Server.setupListenHandle [as _listen2] (net.js:1307:14)
    at listenInCluster (net.js:1355:12)
    at GetAddrInfoReqWrap.doListen [as callback] (net.js:1481:7)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:96:10)
error Command failed with exit code 1.
```

Also, it's an unhandled error, so I can't put it in the `function (err)` part either. So I guess we're stuck with using a note in the README. Unless you think about something else?